papers/2024_Lohrmann_VAMHRI_predictability.bib

```diff
@@ -0,0 +1,8 @@
+@inproceedings{Lohrmann2024HRI,
+    author      = {Lohrmann, Clare and Berg, Ethan and Hayes, Bradley and Roncone, Alessandro},
+    title       = {Improving Robot Predictability via Trajectory Optimization Using a Virtual Reality Testbed},
+    booktitle   = {7th International Workshop on Virtual, Augmented, and Mixed-Reality for Human-Robot Interactions (VAM-HRI)},
+    year        = {2024},
+    organization= {ACM},
+    month       = {Mar}
+}
```

Comment:
you should add the subteams here (see other citations).

Reply:
you should also update the publications page otherwise you will not see it in hiro-group.ronc.one/publications